### ISSUE TITLE
Versioning management do not work

### ISSUE THREAD

**Issue by saulske** [module-federation/medusa#4]
Hi,

Dependency version management do not work. Originally, I heard about it in https://www.youtube.com/watch?v=JkOvNPMo7Os (7:20) demo.

In a current stage, if you clone the project, version management do not work. MF always pick the latest build and medusa configuration settings are ignored.

Example:
* Clone
* Do initial start for dashboard-example and medusa dashboard
* Kill app
* Apply some changes to search component (./MiniSearch.js added text)
* Rebuild and start all apps
* Medusa shows multiple versions with latest one selected by default

Result:
Even medusa shows versions and 0.0.0, 0.1.0 builds are created, remoteEntry.js and Dashboard plugin do not load correct entry. window.search is always primary (no window.search_0_0_0 or window.search_0_1_0) is available. 

Is this a bug or a feature that hasn't been published yet. I would be happy to help fixing it. In general, kudos for this!

**Comment by ScriptedAlchemy** [comment]
version management isnt avaliable as open source

**Comment by ScriptedAlchemy** [comment]
RMM is now fully operational in the close source version of Medusa

**Comment by fatawesome** [comment]
@ScriptedAlchemy hi! Will it be available as a paid feature in near future?

**Comment by ScriptedAlchemy** [comment]
Yeah it will. Working on deploying Medusa as a SaaS but need to redesign the databases to support multiple user isolation.
